azure-pipelines/end-to-end-tests-dir/integrate-install.ps1

```diff
@@ -8,6 +8,7 @@ if (-not $IsLinux -and -not $IsMacOS) {
     # Test msbuild props and targets
     $Script:CurrentTest = "zlib:x86-windows msbuild $iiroot\..."
     Write-Host $Script:CurrentTest
+    Run-Vcpkg @CommonArgs integrate install

```

Comment:
This has machine-wide impacts -- I don't think we should do this in our end-to-end tests by default.

The testing vcxproj files are written to pull in the integrate install files, so they don't require `integrate install`.

I wouldn't be opposed to adding an off-by-default test that _does_ assume it's allowed to mutate global system state.

Reply:
> The testing vcxproj files are written to pull in the integrate install files, so they don't require `integrate install`.

They do not do that. See the disabled test below that talks about needing vcpkg.exe to be located relative to the vcxproj to do that. The only reason this passed before is there happening to be a machine vcpkg somewhere with zlib already installed there.